### DISCUSSION TITLE
(Solve) Not working

### DISCUSSION THREAD

**Discussion by ti-hardikshah** [chris1111/Wireless-USB-Big-Sur-Adapter#44] [Q&A]
Hello, 
I have MacBook Pro (Retina, 15-inch, Mid 2015) & Archer T2U Nano. 
I had tried all steps getting no error !! but still, I am not able to use my Adapter !! 

Any Help ?

**Comment by chris1111** [comment]
This is strange Archer T2U Nano is already test and its a known working adapter
SIP is fully disable?

**Reply by ti-hardikshah** [reply]
Yes, I had followed all steps Except, Uninstall and reinstall again. I will try this and update you tonight.

**Reply by ti-hardikshah** [reply]
Its work after reinstall.

**Comment by tizianoitaly** [comment]
:( 
I can't finish the installation, it reports an error

I confirm SIP is disable

**Reply by ti-hardikshah** [reply]
Did you follow all steps?  try to uninstall and install again.

**Comment by tizianoitaly** [comment]
Good morning, I have tried twice to disinstall and install.
Must the USB device be connetted ?

Tiziano Tanzi
3292371500

Inviato da iPhone

Il giorno 6 gen 2021, alle ore 04:37, Hardik Shah <notifications@github.com> ha scritto:

﻿
Did you follow all steps? try to uninstall and install again.

**Reply by ti-hardikshah** [reply]
Which device are you using? 
After install connect the device to a USB port and follow this
System Preference  > > Security and Privacy >> General tab >> Unlock and give permission for acess 

and done !!

**Comment by tizianoitaly** [comment]
Hi, unfortunately I can't finish the installation. where am i wrong?

best regards





Lemorele Adattatore WiFi USB 3.0 AC1200Mbps USB Bluetooth 5.0 Dual Band 5.8G/2.4G Wireless WiFi Dongle MU-Mimo 5dBi per Windows 10/8.1/8/7/XP,Linux




Il giorno 6 gen 2021, alle ore 10:32, Hardik Shah <notifications@github.com> ha scritto:


Which device are you using?
After install connect the device to a USB port and follow this
System Preference > > Security and Privacy >> General tab >> Unlock and give permission for acess

and done !!
